File: frontend/types/review.ts
export interface ReviewItem {
  review_id: string;
  product_id: string;
  mfg_part_num: string;
  product_name: string;
  field_name: string;
  field?: string;
  issue_type: string;
  severity: 'WARNING' | 'ERROR' | 'CRITICAL';
  current_value: string;
  suggested_value: string;
  reason: string;
  quality_score: number;
  status: 'OPEN' | 'APPROVED' | 'REJECTED' | 'FIXED';
  assigned_to?: string;
  reviewed_at?: string;
  is_demo_data?: boolean;
}

File: backend/app/schemas/review.py
from pydantic import BaseModel
from typing import Optional

class ReviewItemSchema(BaseModel):
    review_id: str
    product_id: str
    mfg_part_num: str
    field: str
    field_name: Optional[str] = None
    issue_type: str
    severity: str  # WARNING, ERROR, CRITICAL
    current_value: str
    suggested_value: str
    reason: str
    quality_score: Optional[float] = 85.0
    product_name: Optional[str] = ""
    status: str  # PENDING, APPROVED, REJECTED, AUTO_FIXED
    created_at: str
